server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -53,14 +52,14 @@ public void toPlainTextShouldRemoveSimpleHtmlTag() {
     @Test
     public void toPlainTextShouldReplaceSkipLine() {
         String html = "<p>This is an<br/>HTML text !</p>";
-        String expectedPlainText = "This is an\nHTML text !\n";
+        String expectedPlainText = "This is an HTML text !";
```

Comment:
Why would we care, we only use it for preview, which do not contains line breaks.

As a proof, JMAP integration tests are stable!